layouts/partials/sitemap.html

```diff
@@ -26,12 +26,12 @@
 
             <div class="card-list">
                 <ul>
-                    <li><a href="{{ .Site.Home.Permalink}}/pages/astronomers/getting_started/">Getting Started</a></li>
-                    <li><a href="https://www.ivoa.net/astronomers/using_the_vo.html">Using the VO</a></li>
-                    <li><a href="https://www.ivoa.net/astronomers/vo_glossary.html">VO Glossary</a></li>
-                    <li><a href="https://www.ivoa.net/astronomers/applications.html">VO Applications</a></li>
-                    <li><a href="https://www.ivoa.net/newsletter/index.html">IVOA newsletter</a></li>
-                    <li><a href="https://www.ivoa.net/astronomers/vo_for_public.html">VO for Students & Public</a></li>
+                    <li><a href="{{ .Site.Home.Permalink}}/astronomers/getting_started/">Getting Started</a></li>
+                    <li><a href="{{ .Site.Home.Permalink}}/astronomers/using_the_vo/">Using the VO</a></li>
+                    <li><a href="{{ .Site.Home.Permalink}}/astronomers/vo_glossary/">VO Glossary</a></li>
+                    <li><a href="{{ .Site.Home.Permalink}}/astronomers/applications/">VO Applications</a></li>
+                    <li><a href="{{ .Site.Home.Permalink}}/newsletter/">IVOA newsletter</a></li>
+                    <li><a href="{{ .Site.Home.Permalink}}/astronomers/vo_for_public/">VO for Students & Public</a></li>
```

Comment:
I think these all have an extra **"/"** between the `{{ .Site.Home.Permlink}}`  and the folder. It works in the browser anyway, but maybe we can remove it.

Reply:
Yes, I have noticed that too. As it does not seem to be an issue, I have not put an high priority to solve this yet. But I can have a quick look now, if you want to.

Reply:
no, definitely no need to prioritise this, let's just keep it in mind